src/database/queries/tag.js

```diff
@@ -0,0 +1,21 @@
+import knex from '../knex'
+import * as _ from './utilities'
+
+const add = attributes =>
+  _.createRecord( 'tag', attributes )
+
+const deleteAll = () =>
```

Comment:
instead of exporting a `deleteAll` which you only need for testing, I would just use knex in your test code to truncate this table beforeEach test